content/millennium/r4/foundation/conformance/structure-definition.md

```diff
@@ -97,6 +97,7 @@ Authorization is not required.
  [`financial-transaction-tendered-amount`]                   | The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
  [`financial-transaction-type`]                              | The classification of the transaction.
  [`formatted-text`]                                          | Link to the formatted text for a note. It is used in cases when the note is not a plain text.
+ [`is-modifiable`]                                              | The extension is present only if activity modifiable is FALSE.
```

Comment:
lets preserve the alignment here.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/697/commits/bc9407c18cb420c2ad537b1a0c384d77f3647634#diff-6d3680a1fcdfd40913793f8a19ab5e89c212edcba4a5b88a7ba7cff44c3af047R100